Source/WebCore/Modules/applepay/ApplePaySessionPaymentRequest.h

```diff
@@ -167,7 +167,38 @@ class ApplePaySessionPaymentRequest {
     const std::optional<ApplePayLaterAvailability>& applePayLaterAvailability() const { return m_applePayLaterAvailability; }
     void setApplePayLaterAvailability(const std::optional<ApplePayLaterAvailability>& applePayLaterAvailability) { m_applePayLaterAvailability = applePayLaterAvailability; }
 #endif
-    
+
+#if PLATFORM(COCOA)
```

Comment:
This isn't necessary.  We already have ENABLE(APPLE_PAY) in this file.

Reply:
Fixed